packages/node-addon-examples/scripts/verify-prebuilds.mts

```diff
@@ -25,6 +25,9 @@ async function verifyAndroidPrebuild(dirent: fs.Dirent) {
   );
   for (const arch of EXPECTED_ANDROID_ARCHS) {
     const archDir = path.join(dirent.parentPath, dirent.name, arch);
+    if (!fs.existsSync(archDir)) {
+      return;
+    }
```

Comment:
This breaks an invariant and shouldn't fail silently.

Reply:
https://github.com/callstackincubator/react-native-node-api/pull/204#issuecomment-3117912710